query/src/main/java/org/infinispan/query/impl/massindex/IndexUpdater.java

```diff
@@ -62,6 +62,7 @@ public void updateIndex(Object key, Object value) {
             DocumentBuilderIndexedEntity docBuilder = entityIndexBinding.getDocumentBuilder();
             final String idInString = keyTransformationHandler.keyToString(key);
             UpdateLuceneWork updateTask = docBuilder.createUpdateWork(
+                  null,
```

Comment:
@Sanne I don't think we have a tenant id in this case, right? Btw. was it good to change this public SPI in that manner?